USHIFT-1459: Add CNCF conformance test scenario (#2149)
openshift/microshift: test/README.md

Runs on the hypervisor. Responsible for launching all of the VMs that
are used in the test step.

> Adding the env variable `PERIODICS` it will create the scenarios defined

Reviewer: Can we make it a longer name? There's too much change of collision this way, I think

Author: Name is changed now and more "non-CI specific".

> Adding the env variable `PERIODICS` it will create the scenarios defined

Reviewer: ```suggestion
When the environment variable `PERIODICS_SCENARIOS` is set, it will create the scenarios defined
```

Author: Updated to the new approach.
